index-community.html

```diff
@@ -50,7 +50,8 @@ <h1 class="cover-heading">Documentation</h1>
              <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Select Version<span class="caret"></span></button>
              <ul class="dropdown-menu" role="menu">
                <li><a href="latest/"><i class="fa fa-arrow-circle-o-right"></i> OKD Latest</a></li>
-               <li><a href="4.13/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.13</a></li>
+               <li><a href="4.13/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.14</a></li>
+               <li><a href="4.14/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.13</a></li>
```

Comment:
Are these backwards?
```suggestion
               <li><a href="4.14/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.14</a></li>
               <li><a href="4.13/"><i class="fa fa-arrow-circle-o-right"></i> OKD 4.13</a></li>
```